### ISSUE TITLE
is it working?

### ISSUE THREAD

**Issue by mhdaslam790** [maaanueeelll/flutter_stripe_terminal_test#1]
i want to integrate terminal in my project and i am looking for package

**Comment by maaanueeelll** [comment]
yep, it's working.
you need to integrate the framework on both sides Android and IOS(you can follow the guide on Stripe terminal)
and use this repo for Flutter.
still I have some issues on IOS when the pos get disconnected, my app crush badly

**Comment by mhdaslam790** [comment]
Alright. Can you guide me trough the instruction to use it? i am bit confused with init.

**Comment by mhdaslam790** [comment]
i am going to test and i will try to fix that crash aswell

**Comment by mhdaslam790** [comment]
no issuw got it .going trough each function step by step helped me.

**Comment by maaanueeelll** [comment]
The guide is well documented

**Comment by mhdaslam790** [comment]
i have changed location and serial in main.dart but still why i am getting only these 3 emulator? it should return the available simulator from my location. ` EVENT DATA {deviceList: [{serialNumber: CHB20SIMULATOR:fab3e01a-378f-489b-a79e-12022ad17510, deviceName: CHIPPER_2X}, {serialNumber: CHB30SIMULATOR:fab3e01a-378f-489b-a79e-12022ad17510, deviceName: STRIPE_M2}, {serialNumber: WPC30SIMULATOR:fab3e01a-378f-489b-a79e-12022ad17510, deviceName: WISEPAD_3}]}
 `

**Comment by mhdaslam790** [comment]
where you have assigned the token? i am unable to find in app side

**Comment by maaanueeelll** [comment]
Nope, the apis for simulation they don’t work with locations, so you get always the same devices.Sent from my iPhoneOn 19 Jan 2023, at 13:07, Mohammad Aslam Ansari ***@***.***> wrote:﻿
i have changed location and serial in main.dart but still why i am getting only these 3 emulator? it should return the available simulator from my location. EVENT DATA {deviceList: [{serialNumber: CHB20SIMULATOR:fab3e01a-378f-489b-a79e-12022ad17510, deviceName: CHIPPER_2X}, {serialNumber: CHB30SIMULATOR:fab3e01a-378f-489b-a79e-12022ad17510, deviceName: STRIPE_M2}, {serialNumber: WPC30SIMULATOR:fab3e01a-378f-489b-a79e-12022ad17510, deviceName: WISEPAD_3}]}
